uint32 id
geometry_msgs/Point centroid
geometry_msgs/Point centroid_vel
geometry_msgs/Point[] members
geometry_msgs/Point[] members_vel
